fabfile/add.py

```diff
@@ -31,6 +43,11 @@
     "./inventory.template": "../{0}/orchestration/inventory/{2}"
 }
 
+SECRETS_PARAMS = [
+    "secret_key",
```

Comment:
Potentially yes, or I could add another method for adding/editing secrets onto specific environments (which was my plan originally).